packages/office-ui-fabric-react/src/components/List/examples/List.Basic.Example.tsx

```diff
@@ -18,10 +18,64 @@ export interface IListBasicExampleState {
   items?: IExampleItem[];
 }
 
+interface IListBasicExampleClassObject {
```

Comment:
In one of your other PRs, if you could change this to `IListBasicExampleClassNames` or just delete the interface entirely (it's really not needed in this case), that would be good.

Reply:
Would also be helpful if in another PR, if you could update `List.doc.tsx` so this example works with export to codepen:

```
// existing import
const ListBasicExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/List/examples/List.Basic.Example.tsx') as string;
// add this import
const ListBasicExampleCodepen = require('!@uifabric/codepen-loader!office-ui-fabric-react/src/components/List/examples/List.Basic.Example.tsx') as string;

// later
    {
      title: 'List of 5000 variable height items',
      code: ListBasicExampleCode,
      codepenJS: ListBasicExampleCodepen, // <-- Add this line
      view: <ListBasicExample items={_cachedItems} />
    },
```